### PR TITLE
ENH: sinh added from todo list #15039

### PR DESCRIPTION
**Pull request by MuhammadNizamani**
I have added sinh in paddle.tensor.math.py

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 1]
```diff
@@ -120,6 +120,12 @@ def ceil(x, name=None):
     return ivy.ceil(x)
 
 
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
```
Comment (AnnaTz) [inline ivy/functional/frontends/paddle/tensor/math.py:124]: The original function's implementation https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/sinh_cn.html#sinh states that the supported 
dtypes are `float32`, `float64` and `float16`

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py [hunk 2]
```diff
@@ -558,4 +560,5 @@ def test_paddle_abs(
         fn_tree=fn_tree,
         on_device=on_device,
         x=x[0],
+        y=x[1],
     )
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py:564]: Please make sure you are not overwriting existing code while adding yours. I think that currently `test_paddle_abs` has been removed.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/paddle/tensor/math.py (hunk 1 above)
Comment (MuhammadNizamani) [reply ivy/functional/frontends/paddle/tensor/math.py:124]: @AnnaTz  you mean  decorater of sinh should with_supported_dtype?

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py (hunk 2 above)
Comment (MuhammadNizamani) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py:564]: You have added this right ? so I should leave it be

File: ivy/functional/frontends/paddle/tensor/math.py (hunk 1 above)
Comment (AnnaTz) [reply ivy/functional/frontends/paddle/tensor/math.py:124]: Yes, exactly.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py (hunk 2 above)
Comment (MuhammadNizamani) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py:564]: Ok I will solve this issue

Comment (MuhammadNizamani) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py:564]: @AnnaTz  just so know I don't overwrite other people's code but for some merge conflict I got confused . sorry for inconvenience.

**Review by AnnaTz — APPROVED** [review]
Looks good now! Merging this 🙂

**Pull request merged**